.github/workflows/ci.yml

```diff
@@ -2,9 +2,14 @@ name: "Terraform CKA Lab VMs"
 
 on:
   push:
-    branches: ["*"]
+    branches: [main]
     paths:
-      - 'proxmox/terraform/**'
+      - 'proxmox/**'
```

Comment:
this should be `proxmox/terraform/**` because other folders in `proxmox` currently only meant to run once and locally (you can checkout README for details)